src/main/java/io/candydoc/plugin/model/DomainModelDto.java

```diff
@@ -17,20 +18,7 @@ public final class DomainModelDto {
   private Collection<DependencyDto> dependencies;
 
   public static DomainModelDto from(DomainModel domainModel) {
-    List<DependencyDto> dependencies = new ArrayList<>();
-
-    domainModel.allowedDependencies().stream()
-        .map(
-            dependency ->
-                DependencyDto.builder().refersTo(dependency.name().value()).allowed(true).build())
-        .forEach(dependencies::add);
-
-    domainModel.forbiddenDependencies().stream()
-        .map(
-            dependency ->
-                DependencyDto.builder().refersTo(dependency.name().value()).allowed(false).build())
-        .forEach(dependencies::add);
-
+    List<DependencyDto> dependencies = domainModel.accept(new AuthorizationDependenciesVisitor());
```

Comment:
I don't think we need a visitor here (depending of what we do to the "deterred" dependencies)